CHANGELOG.md

```diff
@@ -102,6 +102,7 @@
 - [#1179](https://github.com/crypto-org-chain/cronos/pull/1179) Support blocking addresses in mempool.
 - [#1182](https://github.com/crypto-org-chain/cronos/pull/1182) Bump librocksdb to 8.5.3.
 - [#1183](https://github.com/crypto-org-chain/cronos/pull/1183) Avoid redundant logs added from relayer.
+- [#1193](https://github.com/crypto-org-chain/cronos/pull/1193) Make use of binding contract in relayer.
```

Comment:
we'll need to postpone this to next version after `v1.1.0`

Reply:
Got it, just realize some [methods](https://github.com/crypto-org-chain/cronos/pull/1209/files#diff-b93d562e3e27735b0f3311c3b6cf9b278f98f5a1fe03b09cfe232ef752365497R170) seem need to run together for accurate estimate gas